shared/chat/conversation/info-panel/index.desktop.js

```diff
@@ -53,4 +53,81 @@ const InfoPanel = (props: Props) => (
   </Box>
 )
 
-export default InfoPanel
+const ChannelInfoPanel = (props: ChannelInfoPanelProps) => (
+  <Box
+    style={{
+      backgroundColor: globalColors.white,
+      borderLeft: border,
+      borderRight: border,
+      flex: 1,
+      marginTop: -1,
+      overflowY: 'auto',
+    }}
+  >
+    <Box
+      style={{
+        ...globalStyles.flexBoxColumn,
+        alignItems: 'center',
+        borderLeft: border,
+        borderRight: border,
+        marginTop: 20,
+      }}
+    >
+      <Text type="BodySemibold">
+        #{props.channelname}
+      </Text>
+      <Box style={globalStyles.flexBoxRow}>
+        <Avatar teamname={props.teamname} size={16} />
+        <Text style={{marginLeft: globalMargins.xtiny}} type="BodySmall">
+          {props.teamname}
+        </Text>
+      </Box>
+    </Box>
+
+    <Divider style={{marginBottom: 20, marginTop: 20}} />
+
+    <Box
+      style={{...globalStyles.flexBoxColumn, alignItems: 'center', borderLeft: border, borderRight: border}}
+    >
+      <Box style={globalStyles.flexBoxRow}>
+        <Checkbox
+          checked={props.muted}
+          disabled={props.onMuteConversation == null}
+          onCheck={checked => props.onMuteConversation(checked)}
+          label="Mute channel"
+        />
+        <Icon type="iconfont-shh" style={{marginLeft: globalMargins.tiny}} />
+      </Box>
+    </Box>
+
+    <Divider style={{marginBottom: 20, marginTop: 20}} />
+
+    <Box style={{...globalStyles.flexBoxColumn, alignItems: 'center'}}>
+      <Button type="Danger" label="Leave channel" onClick={() => props.onShowBlockConversationDialog()} />
```

Comment:
you can or we can defer that until a different ticket (if so i'd remove the button). it should be a pretty simple api to leave the channel